src/pages/User/components/PullRequests/UserInfo/MotivationalMessage/getMessage.js

```diff
@@ -14,21 +14,27 @@ const messages = [
   "Now you're just showing off!"
 ];
 
+const MONTH_OCTOBER = 9;
+
 /**
  * Get motivational message based on pull requests count.
  *
- * @param {*} pullRequestCount
- * @param {*} otherReposCount
+ * @param {number} pullRequestCount
+ * @param {number} otherReposCount
  * @returns {string}
  */
-function getMessage(pullRequestCount, otherReposCount) {
+function getMessage(pullRequestCount = 0, otherReposCount = 0) {
```

Comment:
No need default values here. We expect them to be provided by the caller.

Reply:
:ok: